src/modules/analysis/HostStatisticsCfg.cpp

```diff
@@ -48,7 +48,7 @@ HostStatisticsCfg* HostStatisticsCfg::create(XMLElement* e)
 HostStatistics* HostStatisticsCfg::createInstance()
```

Comment:
Question: Should we remove `logInterval` completely (HostStatisticsCfg.cpp#30, HostStatisticsCfg.h#46 and ...#52)?